### ISSUE TITLE
Tagged releases

### ISSUE THREAD

**Issue by elfgoh** [RobertCNelson/boot-scripts#53]
(I realised that this issue may not be the right place but I am at a loss where else to start so here goes) I wonder if tagged releases would be a good idea.

The context of this question, I am using console images eg version A[1]. Occasionally a new stable build is updated at [1] eg version B. Is there any visibility as to which additional commits of boot-scripts go into the new stable build? 

The use case is: I do not want to keep changing the console image that I use for customisation, ie keep chasing the latest stable build. So I may want to selectively pull commits to version A.

So ideally, I want a LTS console image, with the option to pull in commits for boot-scripts that are stable fixes via git.

[1] http://elinux.org/Beagleboard:BeagleBoneBlack_Debian#Jessie_Snapshot_console

**Comment by RobertCNelson** [comment]
Not really, most of the time this repo should be safe to just pull.  However with the whole interface rewrite usb0/usb1 so that we no longer need drivers on windows/mac, things kind of became interesting.

Regards,

**Comment by elfgoh** [comment]
So does that mean that this repo is no longer safe to just pull?

This next part is kinda off topic but

- if any, what are the other git repos present in the image?
- if any, are those other git repos safe to pull most of the time?

**Comment by RobertCNelson** [comment]
The change over to dual interfaces happened on March 9th.. So if you are running an image after March 9th, it's safe to pull.. Before March 9th, you just need to make sure you are running a newer kernel (released around march 9th)..

this is the only used repo that is not packaged.

regards,

**Comment by elfgoh** [comment]
Thanks for the information. I am glad that I filed this issue. I had originally intended to use a feb console image. I am under the impression that if I didnt file the image, it would have been difficult to know of similar information that you mentioned.

May I know what are the mechanisms to be informed of such issues when I wish to pull the bootscripts repo?

**Comment by elfgoh** [comment]
May I know which are the other packaged repos?

**Comment by elfgoh** [comment]
Also may I know which are the other packaged repos?

**Comment by RobertCNelson** [comment]
@elfgoh the usb0/usb1 had been the first major change to the boot-scripts startup since we implemented it..

Here's the full list of git repo's we pull in, some need to be packaged, some don't 'need it'..

https://github.com/RobertCNelson/omap-image-builder/blob/master/target/chroot/beagleboard.org-jessie.sh#L208-L324

ps, if "git" isn't installed, these aren't pulled in..

Regards,
